spec/requests/network_routers_spec.rb

```diff
@@ -138,4 +138,17 @@
       expect(response.parsed_body).to include(expected)
     end
   end
+
+  describe 'security groups subcollection' do
+    it "can list a router's security groups" do
+      router = FactoryGirl.create(:network_router)
+      router.security_groups = [FactoryGirl.create(:security_group)]
+      api_basic_authorize(action_identifier(:network_routers, :read, :subcollection_actions, :get))
+
+      get(api_network_router_security_groups_url(nil, router))
+
+      expect(response.parsed_body).to include('subcount' => 1)
```

Comment:
this expectation should probably expect to include the `href` of the `SecurityGroup` too.